ksqldb-common/src/main/java/io/confluent/ksql/util/VertxSslOptionsFactory.java

```diff
@@ -59,6 +60,13 @@ private static JksOptions buildJksOptions(final Buffer buffer, final String pass
     return new JksOptions().setValue(buffer).setPassword(Strings.nullToEmpty(password));
   }
 
+  private static KeyStoreOptions buildBcfksOptions(final String password) {
+    return new KeyStoreOptions()
+        .setType("BCFKS")
+        .setProvider("/opt/confluent/confluent-7.4.0/share/java/kafka/bc-fips-1.0.2.3.jar")
```

Comment:
What if this JAR appears in a different location and/or different version? Can this location be configurable? or why is it hard-coded?